### PR TITLE
Feat: Adds tokens to `Navbar`

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

This PR applies the new [Theming structure](https://github.com/vtex-sites/base.store/pull/407) to `Navbar`: new scoped tokens linked to global ones.

It also refactors and split the navbar-related components as `NavbarSlider` to its own file.

## How does it work?

This PR uses local variables to parameterize the `Navbar` properties, then connects these scoped tokens to the global ones.

Tokens can be visualized in the storybook.

## How to test it?

- The `Navbar` component and its related components should look just as it was before these changes.
- You can check the story on Storybook.

## Checklist

**Changelog**
- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.
- [x] Identified the function or parameter in the PR *- If applicable*. E.g., *`useWindowDimensions` hook*.

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*.
- [ ] For documentation changes, ping @ carolinamenezes, @ PedroAntunesCosta or @ Mariana-Caetano to review and update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,27 +287,6 @@ function Button({ variant, ...props }: Props) {
 ```
 
 Now we have a styled Button component that accepts different variants!! 🎉
-The aforementioned guide works well for UI components. However, components like `Navbar` and `Footer` are more complex and usually don't have variants, since they usually serve a single responsibility on the page. For these cases, you can use [BEM-Style CSS Syntax in SCSS](https://en.bem.info/methodology/) like:
```
Comment (hellofanny) [inline README.md:290]: Thanks for remembering to remove it hihihi! 💯

File: src/components/common/Navbar/Navbar.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+import Navbar, { NavLinks } from '.'
+
+import {
+  TokenTable,
+  TokenRow,
+  TokenDivider,
+} from 'src/../.storybook/components'
+
+<Meta component={Navbar} title="Organisms/Navbar/Default" />
```
Comment (hellofanny) [inline src/components/common/Navbar/Navbar.stories.mdx:14]:
```suggestion
<Meta component={Navbar} title="Organisms/Navbar/Overview" />
```

File: src/components/common/Navbar/navbar-slider.module.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  // Header
+  --fs-navbar-slider-header-padding-bottom           : var(--fs-spacing-2);
+  --fs-navbar-slider-header-button-margin-right      : calc(-1 * var(--fs-spacing-1));
+
+  // Logo
+  --fs-navbar-slider-logo-height                     : 2.375rem;
+  --fs-navbar-slider-logo-padding                    : 0;
+  --fs-navbar-slider-logo-margin-right               : var(--fs-spacing-5);
+  --fs-navbar-slider-logo-img-width                  : 100%;
+  --fs-navbar-slider-logo-img-height                 : 100%;
```
Comment (hellofanny) [inline src/components/common/Navbar/navbar-slider.module.scss:20]: @renatamottam I don't think we need `img-width` and `img-height` here, what do you think?

File: src/components/common/Navbar/navbar.module.scss [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+  // --------------------------------------------------------
+  // Design Tokens for Navbar
+  // --------------------------------------------------------
+
+  // Default properties
+
+  --fs-navbar-padding: 0;
+
+  --fs-navbar-border-bottom-width                             : var(--fs-border-width);
+  --fs-navbar-border-bottom-color                             : var(--fs-color-neutral-2);
```
Comment (hellofanny) [inline src/components/common/Navbar/navbar.module.scss:13]:
```suggestion
  --fs-navbar-border-bottom-color                             : var(--fs-border-color-light);
```

File: src/components/common/Navbar/navbar.module.scss [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+  --fs-navbar-box-shadow                                      : 0 var(--fs-spacing-0) var(--fs-spacing-3) rgb(0 0 0 / 5%);
+
+  // Row
+  --fs-navbar-row-width                                       : 100%;
+
+  // Button Collapse (Search)
+  --fs-navbar-button-collapse-margin-left                     : calc(-1 * var(--fs-spacing-3));
+
+  // Button Cart
+  --fs-navbar-button-cart-notebook-margin-left                : var(--fs-spacing-3);
```
Comment (hellofanny) [inline src/components/common/Navbar/navbar.module.scss:24]:
```suggestion
  --fs-navbar-button-cart-margin-left-notebook                : var(--fs-spacing-3);
```

Comment (hellofanny) [reply src/components/common/Navbar/navbar.module.scss:24]: Pls remember to update all the occurrences, ok?

File: src/components/common/Navbar/navbar.module.scss [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+
+  // Search Input Wrapper
+  --fs-navbar-search-input-wrapper-transition-property        : flex;
+  --fs-navbar-search-input-wrapper-transition-timing          : 0.3s;
+  --fs-navbar-search-input-wrapper-transition-function        : ease-in;
+
+  // Search
+  --fs-navbar-search-button-icon-mobile-width                 : var(--fs-spacing-5);
+  --fs-navbar-search-button-icon-mobile-height                : var(--fs-navbar-search-button-icon-mobile-width);
```
Comment (hellofanny) [inline src/components/common/Navbar/navbar.module.scss:33]:
```suggestion
  --fs-navbar-search-button-icon-width-mobile                : var(--fs-spacing-5);
  --fs-navbar-search-button-icon-height-mobile               : var(--fs-navbar-search-button-icon-mobile-width);
```

File: src/components/common/Navbar/navbar.module.scss [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+  --fs-navbar-search-button-icon-expanded-transition-timing   : 0.3s;
+  --fs-navbar-search-button-icon-expanded-transition-function : ease-in;
+
+  // Header
+  --fs-navbar-header-padding-top                              : var(--fs-spacing-1);
+  --fs-navbar-header-padding-bottom                           : var(--fs-spacing-2);
+  --fs-navbar-header-overflow-x                               : clip;
+
+  --fs-navbar-header-notebook-padding-top                     : var(--fs-spacing-1);
+  --fs-navbar-header-notebook-padding-bottom                  : var(--fs-spacing-1);
```
Comment (hellofanny) [inline src/components/common/Navbar/navbar.module.scss:56]:
```suggestion
  --fs-navbar-header-padding-top-notebook                        : var(--fs-spacing-1);
  --fs-navbar-header-padding-bottom-notebook                 : var(--fs-navbar-header-padding-top-notebook);
```

Comment (hellofanny) [reply src/components/common/Navbar/navbar.module.scss:56]: Pls remember to update all the occurrences, ok?

File: src/components/common/Navbar/navbar.module.scss [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+
+  // Logo
+  --fs-navbar-logo-width                                      : 11rem;
+  --fs-navbar-logo-height                                     : 2.375rem;
+  --fs-navbar-logo-min-height                                 : 2.375rem;
+  --fs-navbar-logo-padding                                    : 0;
+  --fs-navbar-logo-mobile-margin-right                        : var(--fs-spacing-5);
```
Comment (hellofanny) [inline src/components/common/Navbar/navbar.module.scss:63]:
```suggestion
  --fs-navbar-logo-margin-right-mobile                        : var(--fs-spacing-5);
```

Same thing here.

Pls remember to update all the occurrences, ok?

File: src/components/common/Navbar/navbar.module.scss [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+  // Button Collapse (Search)
+  --fs-navbar-button-collapse-margin-left                     : calc(-1 * var(--fs-spacing-3));
+
+  // Button Cart
+  --fs-navbar-button-cart-notebook-margin-left                : var(--fs-spacing-3);
+
+  // Search Input Wrapper
+  --fs-navbar-search-input-wrapper-transition-property        : flex;
+  --fs-navbar-search-input-wrapper-transition-timing          : 0.3s;
+  --fs-navbar-search-input-wrapper-transition-function        : ease-in;
```
Comment (renatamottam) [inline src/components/common/Navbar/navbar.module.scss:29]: I don't think this is needed, we could just use `--fs-navbar-transition-property`

File: src/components/common/Navbar/navbar.module.scss [hunk 10]
```diff
@@ -0,0 +1,214 @@
...
+
+  // Search
+  --fs-navbar-search-button-icon-mobile-width                 : var(--fs-spacing-5);
+  --fs-navbar-search-button-icon-mobile-height                : var(--fs-navbar-search-button-icon-mobile-width);
+
+  // Search variation - Expanded
+  --fs-navbar-search-expanded-width                           : 100%;
+
+  --fs-navbar-search-input-wrapper-expanded-margin-right      : 0.625rem;
```
Comment (renatamottam) [inline src/components/common/Navbar/navbar.module.scss:38]: Maybe `input-expanded` should be together?

File: src/components/common/Navbar/Navbar.stories.mdx [hunk 11]
```diff
@@ -0,0 +1,242 @@
...
+      <h3>Navbar Slider</h3>
+      <p>
+        Displays the <code>NavLinks</code> and some features using{' '}
+        <code>SlideOver</code> component.
+      </p>
+    </article>
+  </article>
+  <article>
+    <div style={{ height: '470px' }}>
+      <a href="?path=/docs/organisms-navbar-navlinks--overview-default">
```
Comment (hellofanny) [inline src/components/common/Navbar/Navbar.stories.mdx:227]: Hahah too big, maybe we can just display some links here? 🤔 

image

File: src/components/common/Navbar/navbar-slider.module.scss (hunk 3 above)
Comment (renatamottam) [reply src/components/common/Navbar/navbar-slider.module.scss:20]: Agreed!

Comment (eduardoformiga) [reply src/components/common/Navbar/navbar-slider.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/b8b5895594d8b5f0f01a78395a12219191888ed0).

File: src/components/common/Navbar/navbar.module.scss (hunk 4 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/e4170adbaf2599a5a78a505664ad6bdbf5d03c79)

File: src/components/common/Navbar/navbar.module.scss (hunk 5 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/0d55c686103ff7d6e80d45b4ce14c06221086081).

File: src/components/common/Navbar/navbar.module.scss (hunk 6 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/9e560bb64dcc3627835f6795e777601de17476d6).

File: src/components/common/Navbar/navbar.module.scss (hunk 7 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/9e560bb64dcc3627835f6795e777601de17476d6).

File: src/components/common/Navbar/Navbar.stories.mdx [hunk 12]
```diff
@@ -0,0 +1,242 @@
...
+
+---
+
+## Nested Elements
+
+### Row
+
+<TokenTable>
+  <TokenRow token="--fs-navbar-row-width" value="100%" />
+</TokenTable>
```
[the remark this answers is not in the document]
Comment (eduardoformiga) [reply src/components/common/Navbar/Navbar.stories.mdx]: Nice! done [here](https://github.com/vtex-sites/nextjs.store/pull/142#:~:text=Groups%20row%20token-,99083ad,-Merge%20state).

File: src/components/common/Navbar/navbar.module.scss (hunk 9 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: I wonder if I understand well. Changes were done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/3bf94796037f15383350f1a8e7467647a5bc5efc). Is that what you mean?

File: src/components/common/Navbar/Navbar.stories.mdx (hunk 11 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/Navbar.stories.mdx:208]: Here I see in that way hahaha. 
Where do you see this huge image in Navlinks?

Screen Shot 2022-06-28 at 19 33 05

Agree that the space is too big to show only some links, maybe I can show the desktop and mobile together. What do you think?

File: src/components/common/Navbar/navbar.module.scss (hunk 10 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss:34]: I was with that in mind as well. But we have the `search-input-wrapper` element and a kind of variation called `expanded`. Thus, I prefer to keep it as `--fs-navbar-search-input-wrapper-expanded...` but we can discuss more.

File: src/components/common/Navbar/Navbar.stories.mdx (hunk 11 above)
Comment (renatamottam) [reply src/components/common/Navbar/Navbar.stories.mdx:208]: I created a new `SectionList` component for stories on the #139, you could try to use it after the merge :)

Comment (eduardoformiga) [reply src/components/common/Navbar/Navbar.stories.mdx:208]: Let's use it 😉

File: src/components/common/Navbar/Navbar.stories.mdx [hunk 13]
```diff
@@ -0,0 +1,206 @@
...
+
+## Compound Components
+
+<section className="sbdocs-section">
+  <article>
+    <div style={{ height: '470px' }}>
+      <a href="?path=/docs/organisms-navbar-navbarslider--overview-default">
+        <img
+          width="245px"
+          src="https://vtexhelp.vtexassets.com/assets/docs/src/navlinks___52dff993e0e04dea5d02c1e1a5bbf4e3.png"
```
Comment (ArthurTriis1) [inline src/components/common/Navbar/Navbar.stories.mdx:177]: Can we put the component here instead the image?

File: src/components/common/Navbar/navbar.module.scss [hunk 14]
```diff
@@ -0,0 +1,199 @@
...
+  --fs-navbar-search-button-icon-width-mobile            : var(--fs-spacing-5);
+  --fs-navbar-search-button-icon-height-mobile           : var(--fs-navbar-search-button-icon-width-mobile);
+
+  // Search Expanded
+  --fs-navbar-search-expanded-input-wrapper-margin-right : 0.625rem;
+  --fs-navbar-search-expanded-input-wrapper-margin-left  : var(--fs-spacing-9);
+
+  --fs-navbar-search-expanded-input-width                : calc(100% - var(--fs-spacing-7));
+
+  --fs-navbar-search-expanded-button-icon-margin-right   : -4.063rem;
```
Comment (ArthurTriis1) [inline src/components/common/Navbar/navbar.module.scss:38]: I think this already existed, but do you know where come from this number?

File: src/components/common/Navbar/navbar.module.scss (hunk 9 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: changes implemented [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/9b4bd25dcc3eb8016c59436b5381ec3bfb42be8b) and [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/2aad36dad44008cff7d19b17f115ad61d60af2df).

File: src/components/common/Navbar/navbar.module.scss (hunk 10 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: changes implemented [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/9b4bd25dcc3eb8016c59436b5381ec3bfb42be8b) and [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/2aad36dad44008cff7d19b17f115ad61d60af2df). What do you think?

File: src/components/common/Navbar/navbar.module.scss (hunk 14 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss:38]: yeah, this already existed. 
I guess this was the number that fits well when the Search input icon was being tested.

File: src/components/common/Navbar/navlinks.module.scss [hunk 15]
```diff
@@ -0,0 +1,81 @@
+@import "src/styles/scaffold";
+
+.fs-navlinks {
+  // --------------------------------------------------------
+  // Design Tokens for Navlinks
+  // --------------------------------------------------------
+
+  --fs-navlinks-notebook-padding-top             : var(--fs-spacing-1);
```
Comment (hellofanny) [inline src/components/common/Navbar/navlinks.module.scss:8]:
```suggestion
  --fs-navlinks-padding-top-notebook             : var(--fs-spacing-1);
```

File: src/components/common/Navbar/navlinks.module.scss [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+
+  --fs-navlinks-mobile-border-top-width          : var(--fs-border-width);
+  --fs-navlinks-mobile-border-top-color          : var(--fs-color-neutral-2);
+
+  --fs-navlinks-mobile-border-bottom-width       : var(--fs-border-width);
+  --fs-navlinks-mobile-border-bottom-color       : var(--fs-color-neutral-2);
```
Comment (hellofanny) [inline src/components/common/Navbar/navlinks.module.scss:14]:
```suggestion
  --fs-navlinks-border-top-width-mobile         : var(--fs-border-width);
  --fs-navlinks-border-top-color-mobile          : var(--fs-color-neutral-2);

  --fs-navlinks-border-bottom-width-mobile       : var(--fs-navlinks-border-top-width-mobile);
  --fs-navlinks-border-bottom-color-mobile        : var(--fs-navlinks-border-top-color-mobile);
```

Comment (hellofanny) [reply src/components/common/Navbar/navlinks.module.scss:14]: Can you apply this for all the occurrence please?

File: src/components/common/Navbar/navlinks.module.scss [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+
+  // Link
+  --fs-navlinks-link-mobile-width                : 100%;
```
Comment (hellofanny) [inline src/components/common/Navbar/navlinks.module.scss:17]: Same thing for the following tokens. We are applying the `breakpoint` by the end of the token.

**Comment by hellofanny** [comment]
@eduardoformiga I think you forgot to update `navlinks.module.scss` tokens (can you rename them, so the `breakpoint` part is by the end?)

**Comment by eduardoformiga** [comment]
> 

You are right!
I left it behind. Done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/f8e4aad43010ac86f472705ac528cf4e29bf25f1). Thanks! 💛

File: src/components/common/Navbar/navbar-slider.module.scss [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+  --fs-navbar-slider-logo-height                     : 2.375rem;
+  --fs-navbar-slider-logo-padding                    : 0;
+  --fs-navbar-slider-logo-margin-right               : var(--fs-spacing-5);
+
+  // Footer
+  --fs-navbar-slider-footer-padding-top              : var(--fs-spacing-2);
+  --fs-navbar-slider-footer-margin-top               : var(--fs-spacing-2);
+
+  --fs-navbar-slider-footer-signin-link-width        : fit-content;
+  --fs-navbar-slider-footer-signin-link-padding-left : 0;
```
Comment (hellofanny) [inline src/components/common/Navbar/navbar-slider.module.scss:25]: @renatamottam do you think we need those tokens?

**Review by hellofanny — APPROVED** [review]
LGTM! 🚀 🚀  Let's see if @renatamottam has any further considerations.

File: src/components/common/Navbar/navbar-slider.module.scss (hunk 18 above)
Comment (renatamottam) [reply src/components/common/Navbar/navbar-slider.module.scss:25]: nope!

**Review by renatamottam — COMMENTED** [review]
Left some comments below as an effort to simplify these tokens :)

File: src/components/common/Navbar/navbar.module.scss [hunk 19]
```diff
@@ -0,0 +1,199 @@
...
+  // Default properties
+
+  --fs-navbar-padding                                    : 0;
+
+  --fs-navbar-border-bottom-width                        : var(--fs-border-width);
+  --fs-navbar-border-bottom-color                        : var(--fs-border-color-light);
+
+  --fs-navbar-box-shadow                                 : 0 var(--fs-spacing-0) var(--fs-spacing-3) rgb(0 0 0 / 5%);
+
+  --fs-navbar-row-width                                  : 100%;
```
[the remark this answers is not in the document]
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/1027d3c6aad26e28a6ed9d35d35ef8d704f495e6).

File: src/components/common/Navbar/navbar.module.scss [hunk 20]
```diff
@@ -0,0 +1,199 @@
...
+
+  --fs-navbar-transition-timing                          : var(--fs-transition-timing);
+  --fs-navbar-transition-function                        : var(--fs-transition-function);
+
+  // Button Collapse (Search)
+  --fs-navbar-button-collapse-margin-left                : calc(-1 * var(--fs-spacing-3));
```
[the remark this answers is not in the document]
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/9760ee30a005cd54a324d0b5dfdc5c3ffda1ab9a).

File: src/components/common/Navbar/navbar.module.scss [hunk 21]
```diff
@@ -0,0 +1,199 @@
...
+
+  // Button Cart
+  --fs-navbar-button-cart-margin-left-notebook           : var(--fs-spacing-3);
```
[the remark this answers is not in the document]
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/773d5d3a3dd8eda0801f413c39e3b1aa14f3987b).

File: src/components/common/Navbar/navbar.module.scss [hunk 22]
```diff
@@ -0,0 +1,199 @@
...
+  --fs-navbar-search-expanded-input-wrapper-margin-left  : var(--fs-spacing-9);
+
+  --fs-navbar-search-expanded-input-width                : calc(100% - var(--fs-spacing-7));
+
+  --fs-navbar-search-expanded-button-icon-margin-right   : -4.063rem;
+
+  // Header
+  --fs-navbar-header-padding-top                         : var(--fs-spacing-1);
+  --fs-navbar-header-padding-bottom                      : var(--fs-spacing-2);
+  --fs-navbar-header-overflow-x                          : clip;
```
[the remark this answers is not in the document]
Comment (eduardoformiga) [reply src/components/common/Navbar/navbar.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/b76f179863cd84c3a45129827f55f059239196b0).

File: src/components/common/Navbar/navlinks.module.scss [hunk 23]
```diff
@@ -0,0 +1,80 @@
...
+
+  --fs-navlinks-padding-top-notebook             : var(--fs-spacing-1);
+
+  --fs-navlinks-border-top-width-mobile          : var(--fs-border-width);
+  --fs-navlinks-border-top-color-mobile          : var(--fs-color-neutral-2);
+  --fs-navlinks-border-bottom-width-mobile       : var(--fs-navlinks-border-top-width-mobile);
+  --fs-navlinks-border-bottom-color-mobile       : var(--fs-navlinks-border-top-color-mobile);
+
+  // Link
+  --fs-navlinks-link-width-mobile                : 100%;
```
[the remark this answers is not in the document]
Comment (eduardoformiga) [reply src/components/common/Navbar/navlinks.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/74a14e8da743cd8b041b2934e293272a74e7d133).

File: src/components/common/Navbar/navbar-slider.module.scss (hunk 18 above)
Comment (hellofanny) [reply src/components/common/Navbar/navbar-slider.module.scss:25]: @eduardoformiga what do you think?

Comment (eduardoformiga) [reply src/components/common/Navbar/navbar-slider.module.scss]: Agree with you! done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/77e7c0ede36cc95f3f499a5693f8e98fc85ca147). Thanks!

File: src/components/common/Navbar/navlinks.module.scss [hunk 24]
```diff
@@ -0,0 +1,78 @@
...
+
+.fs-navlinks {
+  // --------------------------------------------------------
+  // Design Tokens for Navlinks
+  // --------------------------------------------------------
+
+  --fs-navlinks-padding-top-notebook             : var(--fs-spacing-1);
+
+  --fs-navlinks-border-top-width-mobile          : var(--fs-border-width);
+  --fs-navlinks-border-top-color-mobile          : var(--fs-color-neutral-2);
```
Comment (renatamottam) [inline src/components/common/Navbar/navlinks.module.scss:11]:
```suggestion
  --fs-navlinks-border-top-color-mobile          : var(--fs-border-color-light);
```

File: src/components/common/Navbar/NavLinks.stories.mdx [hunk 25]
```diff
@@ -0,0 +1,104 @@
...
+  />
+  <TokenDivider />
+  <TokenRow
+    token="--fs-navlinks-border-bottom-width-mobile"
+    value="var(--fs-navlinks-border-top-width-mobile)"
+  />
+  <TokenRow
+    token="--fs-navlinks-border-bottom-color-mobile"
+    value="var(--fs-navlinks-border-top-color-mobile)"
+    isColor
```
Comment (renatamottam) [inline src/components/common/Navbar/NavLinks.stories.mdx:67]:
```suggestion
    token="--fs-navlinks-border-bottom-color-mobile"
    value="var(--fs-navlinks-border-top-color-mobile)"
    globalValue="var(--fs-border-color-light)"
    isColor
```

File: src/components/common/Navbar/NavLinks.stories.mdx [hunk 26]
```diff
@@ -0,0 +1,104 @@
...
+    value="var(--fs-spacing-1)"
+  />
+  <TokenDivider />
+  <TokenRow
+    token="--fs-navlinks-border-top-width-mobile"
+    value="var(--fs-border-width)"
+  />
+  <TokenRow
+    token="--fs-navlinks-border-top-color-mobile"
+    value="var(--fs-color-neutral-2)"
```
Comment (renatamottam) [inline src/components/common/Navbar/NavLinks.stories.mdx:56]:
```suggestion
    value="var(--fs-border-color-light)"
```

**Review by renatamottam — APPROVED** [review]
Nice one! Approved with tiny suggestions :)

File: src/components/common/Navbar/navlinks.module.scss (hunk 24 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/navlinks.module.scss:11]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/43b689f875adcb7ae1fdca68c21531e41a531b2c).

File: src/components/common/Navbar/NavLinks.stories.mdx (hunk 26 above)
Comment (eduardoformiga) [reply src/components/common/Navbar/NavLinks.stories.mdx]: done [here](https://github.com/vtex-sites/nextjs.store/pull/142/commits/20ea86a6a3da721c4ad2cb9a1e66a1fb066d189e).

**Pull request merged**